plugins/UsersManager/API.php

```diff
@@ -1297,18 +1297,16 @@ private function isUserTheOnlyUserHavingSuperUserAccess($userLogin)
      * If the username/password combination is incorrect an invalid token will be returned.
      *
      * @param string $userLogin Login or Email address
-     * @param string $md5Password hashed string of the password (using current hash function; MD5-named for historical reasons)
+     * @param string $passwordConfirmation hashed string of the password (using current hash function; MD5-named for historical reasons)
```

Comment:
guess that comment needs to be adjusted, or does it still need to be a hashed password?

Reply:
updated